bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/database/DBOpenHelper.kt

```diff
@@ -1,49 +1,59 @@
 package io.bucketeer.sdk.android.internal.database
 
 import android.content.Context
+import androidx.annotation.VisibleForTesting
 import androidx.sqlite.db.SupportSQLiteDatabase
 import androidx.sqlite.db.SupportSQLiteOpenHelper
 import androidx.sqlite.db.framework.FrameworkSQLiteOpenHelperFactory
 import io.bucketeer.sdk.android.internal.database.migration.Migration1to2
+import io.bucketeer.sdk.android.internal.database.migration.Migration2to3
 import io.bucketeer.sdk.android.internal.evaluation.db.EvaluationEntity
 import io.bucketeer.sdk.android.internal.event.EventEntity
 
 class OpenHelperCallback : SupportSQLiteOpenHelper.Callback(VERSION) {
 
   companion object {
     const val FILE_NAME = "bucketeer.db"
-    const val VERSION = 2
+    const val VERSION = 3
+
+    @VisibleForTesting
+    fun v2Schema(db: SupportSQLiteDatabase) {
```

Comment:
To avoid duplicate code, I put the schema in this static function so we can reuse it in the [tests](https://github.com/bucketeer-io/android-client-sdk/pull/79/files#diff-9e9eebfdac59b75de3adc119068791edf454ffff72a7cddfd427a804bbadb7a7R107).
If there is a better of doing this, please let me know.